src/main/scala/uk/gov/hmrc/SbtSassCompiler.scala

```diff
@@ -120,7 +123,16 @@ object SbtSassCompiler extends AutoPlugin {
             (anySassFileChanged: Boolean, _) =>
               if (anySassFileChanged || (sassFilesFound.nonEmpty && existingCssFromTargetDirectory.isEmpty)) {
                 logger.info("sbt-sass-compiler: Uncompiled Sass files found")
-                compileSassToCssFiles()
+                try {
+                  compileSassToCssFiles()
+                } catch {
+                  case ise: IllegalStateException => {
+                    logger.error("sbt-sass-compiler: Sass compilation failed: " + ise)
+                    logger.info("sbt-sass-compiler: attempting to recover by removing the path and re-running SassCompilerFactory.bundled()")
+                    delete(new File(s"/tmp/de.larsgrefer.sass.embedded.connection.BundledPackageProvider/${sassEmbeddedHostVersion}/dart-sass/1.83.4/dart-sass/"))
```

Comment:
oh! @TimothyFothergill I messed up on this, this path needs to have a dynamic prefix of java.io.tmpdir

like https://github.com/larsgrefer/dart-sass-java/blob/d657dc995b8ea2df2908dcd8044d3756c028e83f/sass-embedded-host/src/main/java/de/larsgrefer/sass/embedded/connection/DartSassPackageProvider.java#L75

because on unix it's /tmp but on mac it's more dynamic, here's what it is for me:

![CleanShot 2025-03-17 at 14 50 19@2x](https://github.com/user-attachments/assets/4b3f7cfb-15cf-4fad-ba1e-e15797c31583)

luckily sbt io delete that we're using on the file fails silently if it doesn't exist or there is an error so won't impact anyone - but won't work for anyone not on linux at the moment